### ISSUE TITLE
v2.0.0 Product FAQ (product_faq.md) Doc Update

### ISSUE THREAD

**Issue by lanisyutin** [milvus-io/milvus-docs#1208]
### Is there an existing issue for this?

- [X] I have searched the existing issues.

### Issue

Now threre is a paragraph in the doc:

_Why do Euclidean distance (L2) and inner product (IP) return different results?
For normalized vectors, Euclidean distance (L2) is mathematically equivalent to inner product (IP). If these similarity metrics return different results, check to see if your vectors are normalized_

I think it is incorrect. Simply run this code:
```
from scipy.spatial import distance
from sklearn.preprocessing import normalize
n = 2
vectors = [[random.random() for _ in range(128)] for _ in range(n)]
normalized_vectors = normalize(vectors)

assert np.allclose(1, np.linalg.norm(normalized_vectors, axis=1)) 

ip = normalized_vectors[0].dot(normalized_vectors[1])
ed = distance.euclidean(normalized_vectors[0], normalized_vectors[1])

assert np.allclose(ip, ed), f"Inner product is not equals to Euclidean distance! Diff: {abs(ip - ed)}" # <--- it fails
```

### Suggestion

Remove it from doc or clarify

**Comment by lanisyutin** [comment]
I thought about it a little more. It seems that this does not mean mathematical equivalence, but preservation of order. Because all norms in the finite-dimensional real vector space are equivalent.[Riss theorem](https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=&ved=2ahUKEwi38bOjq9H0AhUKt4sKHbMhCwAQFnoECAwQAw&url=https%3A%2F%2Fen.wikipedia.org%2Fwiki%2FNorm_(mathematics)%23%3A~%3Atext%3DIf%2520the%2520vector%2520space%2520is%2Cnot%2520need%2520to%2520be%2520distinguished.&usg=AOvVaw3sWaBCY2SnGA40LFxM70k3). But not all equivalent ones keep the order

**Comment by ss892714028** [comment]
$$
d(A, B)=\sqrt{\sum_{1}^{n}\left(a_{i}-b_{i}\right)^{2}}=\sqrt{\sum_{1}^{n}\left(a_{i}^{2}-2 a_{i} b i+b_{i}^{2}\right)}=\sqrt{\sum_{1}^{n} a_{i}^{2}-2 \sum_{1}^{n} a_{i} b i+\sum_{1}^{n} b_{i}^{2}}=\sqrt{2-2 \times p(A, B)}
$$

**Comment by ss892714028** [comment]
![image](https://user-images.githubusercontent.com/34635663/145016055-b9af6578-b069-46d6-a5e0-24f3c676eb60.png)
d(A, B) represents the L2 Norm and p(A, B) represents the inner product. Assuming vectors A and B are normalized.

**Comment by ss892714028** [comment]
I am closing this issue since it has been inactive for 20 days. If you have further questions, please open another issue. @lanisyutin
